phylogenetic/build-configs/nextstrain-automation/deploy.smk

```diff
@@ -0,0 +1,15 @@
+"""
+This part of the workflow handles automatic deployments of the zika build.
```

Comment:
```suggestion
This part of the workflow handles automatic deployments of the dengue build.
```

Reply:
@victorlin @joverlee521 , could we generalize this and just say "pathogen" build?

This way there's less to trip-over when applying these rules to other pathogens of interest.

Reply:
Thanks, submitted as a hotfix: https://github.com/nextstrain/dengue/commit/b2c36557d6c6b209d512dec7da10a4f0d8cc7b70

Reply:
Good idea, +1 for less pathogen-specific stuff.